llvm/lib/SYCLLowerIR/ESIMD/LowerESIMDKernelAttrs.cpp

```diff
@@ -25,8 +25,8 @@ bool isInvokeSimdBuiltinCall(const CallInst *CI) {
   return F && F->getName().startswith(esimd::INVOKE_SIMD_PREF);
 }
 
-// Checks the use of a function address being stored in a memory.
-// Returns false if the function address is used as an argument for
+// Checks the use of a function.
```

Comment:
There is no "function" per se in any of the parameters. So the comment is not clear.